storage_service/metsrw/exceptions.py

```diff
@@ -0,0 +1,13 @@
+"""
```

Comment:
Why not specify this as a pip dependency instead of vendoring it? Pip can install from git, and we can start working on getting it into PyPI. See https://github.com/artefactual/automation-tools/blob/dev/dataverse/requirements/base.txt#L4